build-aux/m4/bitcoin_secp.m4

```diff
@@ -9,6 +9,25 @@ AC_LINK_IFELSE([AC_LANG_PROGRAM([[
 AC_MSG_RESULT([$has_64bit_asm])
 ])
 
+AC_DEFUN([SECP_ARM32_ASM_CHECK], [
+  AC_MSG_CHECKING(for ARM32 assembly availability)
+  SECP_ARM32_ASM_CHECK_CFLAGS_saved_CFLAGS="$CFLAGS"
+  CFLAGS="-x assembler"
+  AC_COMPILE_IFELSE([[
```

Comment:
I think we want to need to run the linker here, because checking the compiler may not be enough when LTO is enabled, see https://github.com/bitcoin-core/secp256k1/pull/1104.

Reply:
Updated.